modules/distribution/carbon-home/conf/manager/deployment.yaml

```diff
@@ -290,4 +290,7 @@ deployment.config:
   datasource: SP_MGT_DB           # define a mysql datasource in datasources and refer it from here.
   minResourceCount: 1
   bootstrapURLs: localhost:9092   # kafka urls
-  zooKeeperURLs: localhost:2181   # zookeeper urls
+  zooKeeperConfig:
```

Comment:
This change breaks backward compatibility with SP-4.0.0 & SP-4.1.0. Shall we try an approach that preserves backward compatibility.

Reply:
The old config is not removed but deprecated so that nothing breaks.

Reply:
Let's also put a log so that user knows when they are using deprecated configs

Reply:
Will add a warn log and update in carbon-analytics

Reply:
We can add that log when we consume `zooKeeperConfig`. Check whether `zooKeeperURLs` is not null, and if so print a WARN log.

Reply:
Log added in wso2/cabon-analytics#1258